it/utils/ComponentSpecHelper.scala

```diff
@@ -73,29 +78,25 @@ trait ComponentSpecHelper extends AnyWordSpec with Matchers
     super.beforeEach()
   }
 
-  def get[T](uri: String): WSResponse = {
+  def get[T](uri: String): WSResponse =
     await(buildClient(uri).withHttpHeaders("Authorization" -> "Bearer123").get)
-  }
 
-  def post[T](uri: String)(body: T)(implicit writes: Writes[T]): WSResponse = {
+  def post[T](uri: String)(body: T)(implicit writes: Writes[T]): WSResponse =
     await(
       buildClient(uri)
         .withHttpHeaders("Content-Type" -> "application/json", "Authorization" -> "Bearer123")
         .post(writes.writes(body).toString())
     )
-  }
 
-  def put[T](uri: String)(body: T)(implicit writes: Writes[T]): WSResponse = {
+  def put[T](uri: String)(body: T)(implicit writes: Writes[T]): WSResponse =
     await(
       buildClient(uri)
         .withHttpHeaders("Content-Type" -> "application/json", "Authorization" -> "Bearer123")
         .put(writes.writes(body).toString())
```

Comment:
I don't think we can remove the implicit value writes because the full definition of Json.toBody is 

def toJson[T](o: T)(implicit tjs: Writes[T])

However, it does allow us to dispense with "toString" which I like.